isValueEven.js

```diff
@@ -1,7 +1,30 @@
-function isEvenValue(number) {
-  if (number / 2 === 1) {
+function isValueEven(number) {
+  isInputNumber(number);
+  isInputAnInteger(number);
+  isInputZero(number);
+  if (number % 2 === 0) {
     return true;
-  } else {
-    return false;
   }
+  return false;
 }
+
+isInputNumber = (input) => {
+  if (typeof input !== "number" || isNaN(input) === true) {
+    throw new Error("You must type number");
+  }
+};
+
+isInputAnInteger = (input) => {
```

Comment:
See why we couldn't check numbers other than integers? We can check if 3.3425 is even or not. (We both know it is not) But we can still do it :)